lib/wp2ghost.js

```diff
@@ -119,10 +119,18 @@ exports.fromStream = function(stream) {
 
       var date;
 
+      date = item['wp:post_date_gmt'];
+      if (date == null) {
```

Comment:
The original check for 0000-00-00 00:00:00 was only against `post_date_gmt` , should it also check `post_date`?